core-blocks/gallery/edit.js

```diff
@@ -246,7 +236,7 @@ class GalleryEdit extends Component {
 					</PanelBody>
 				</InspectorControls>
 				{ noticeUI }
-				<ul className={ `${ className } align${ align } columns-${ columns } ${ imageCrop ? 'is-cropped' : '' }` }>
+				<ul className={ `${ className } columns-${ columns } ${ imageCrop ? 'is-cropped' : '' }` }>
```

Comment:
Did you check if that doesn't break anything? It seems like it exists here for a reason.

Reply:
Let's bring it back. I'm seeing a  slight change in the UI when this class is missing.